peachjam/migrations/0115_copy_and_delete_old_order_outcome_field_data.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.19 on 2024-01-17 09:20
+
+from django.db import migrations, models
+
+
+def copy_data(apps, schema_editor):
+    Judgment = apps.get_model("peachjam", "Judgment")
+
+    for judgment in Judgment.objects.all().iterator(chunk_size=100):
```

Comment:
```suggestion
    for judgment in Judgment.objects.filter(order_outcome__isnull=False).iterator(chunk_size=100):
```

otherwise we have to iterate and update 70,000 judgments that aren't changing.

Reply:
Also, looks like the migration needs to be tweaked:

```
  Applying peachjam.0115_copy_and_delete_old_order_outcome_field_data...Traceback (most recent call last):
  File "/Users/greg/coding/laws.africa/peachjam/.env/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "orderoutcome_id" of relation "peachjam_judgment_order_outcomes" violates not-null constraint
```